examples/horizontal_auto_scale.py

```diff
@@ -0,0 +1,295 @@
+        >>> result = thread.get()
+
+        :param async_req bool: execute request asynchronously
+        :param str name: name of the hpa (required)
+        :param str namespace: object name and auth scope, such as for teams and projects (required)
+        :param object body: (required)
+        :param str pretty: If 'true', then the output is pretty printed.
+        :param str dry_run: When present, indicates that modifications should not be persisted. An invalid or unrecognized dryRun directive will result in an error response and no further processing of the request. Valid values are: - All: all dry run stages will be processed
+        :param str field_manager: fieldManager is a name associated with the actor or entity that is making these changes. The value must be less than or 128 characters long, and only contain printable characters, as defined by https://golang.org/pkg/unicode/#IsPrint. This field is required for apply requests (application/apply-patch) but optional for non-apply patch types (JsonPatch, MergePatch, StrategicMergePatch).
+        :param bool force: Force is going to \"force\" Apply requests. It means user will re-acquire conflicting fields owned by other people. Force flag must be unset for non-apply patch requests.
+        :param _return_http_data_only: response data without head status code
+                                       and headers
+        :param _preload_content: if False, the urllib3.HTTPResponse object will
+                                 be returned without reading/decoding response
+                                 data. Default is False.
+        :param _request_timeout: timeout setting for this request. If one
+                                 number provided, it will be total request
+                                 timeout. It can also be a pair (tuple) of
+                                 (connection, read) timeout
+        :param _ignore_params : defaults to []
+        :param _request_type: defaults to GET.
+        :return: **
+                 If the method is called asynchronously,
+                 returns the request thread.
+        """
+
+        local_var_params = locals()
+        all_params = [
+            'name',
+            'namespace',
+            'body',
+            'pretty',
+            'dry_run',
+            'field_manager',
+            'force'
+        ]
+        all_params.extend(
+            [
+                'async_req',
+                '_return_http_data_only',
+                '_preload_content',
+                '_request_timeout',
+                '_request_type',
+                '_ignore_params'
+            ]
+        )
+
+        for key, val in six.iteritems(local_var_params['kwargs']):
+            if key not in all_params:
+                raise ApiTypeError(
+                    "Got an unexpected keyword argument '%s'"
+                    " to method patch_namespaced_hpa" % key
```

Comment:
may not be patch_namespaced_hpa

Reply:
Setting it to name of common function which is used for all requests : 'hpa_namespaced_with_http_info'